tracer/src/Datadog.Trace/ClrProfiler/ClrNames.cs

```diff
@@ -45,5 +45,7 @@ internal static class ClrNames
         public const string GenericParameterTask = "System.Threading.Tasks.Task`1<T>";
         public const string ObjectTask = "System.Threading.Tasks.Task`1<System.Object>";
         public const string Int32Task = "System.Threading.Tasks.Task`1<System.Int32>";
+
+        public const string HttpConfigurationAction = "System.Action`1<System.Web.Http.HttpConfiguration>";
```

Comment:
Are you using this anywhere? (I couldn't find it.)

Reply:
Whoops this is no longer needed, good catch! Removed in b5afa0200db546dca90dc571b52065c20e6d4d83